pkg/config/environment_detection.go

```diff
@@ -114,7 +114,9 @@ func DetectFeatures() {
 			}
 		}
 
-		log.Infof("Features detected from environment: %v", newFeatures)
+		if len(newFeatures) > 0 {
```

Comment:
I think we should remove the `if` to also get the log line when any feature is enable